cinch/install-rhel7.yml

```diff
@@ -7,7 +7,7 @@
 # This playbook was tested with Ansible 1.8.4.
 
 - name: install cinch into a virtualenv on RHEL7
-  hosts: localhost
+  connection: local
```

Comment:
I'm not sure this change was correct. We don't want to execute this a dozen times locally if someone passes in an inventory script with a dozen different hosts. We really do only want this to execute on localhost.